backend/DAO/placementDAO.js

```diff
@@ -134,7 +134,19 @@ module.exports = {
 
     },
 
-    getPlacementSkillsByID: (placementID) => {
+    getPlacements: () => {
+        return database('placements')
+            .select('id',
+            'position',
+            'working_hours',
+            'start_period',
+            'end_period',
+            'salary',
+            'institution',
+            'description_role',
+            'major');
```

Comment:
we don't have majors anymore in the placements table; we have the table "placement_has_major"

Reply:
I eliminated those lines